src/encoded/schemas/changelogs/software.md

```diff
@@ -1,4 +1,7 @@
 ## Changelog for software.json
+### Schema version 11
```

Comment:
As the code is now, this doesn't require a whole new schema version and is just a minor change. If we are making the "software_type" and "used_by" properties required that would warrant a new schema version and corresponding upgrade.